tidy3d/components/mode_spec.py

```diff
@@ -433,7 +433,7 @@ def sampling_points(self, freqs: FreqArray) -> FreqArray:
         >>> interp_spec = ModeInterpSpec.cheb(num_points=10)
         >>> sampling_freqs = interp_spec.sampling_points(freqs)
         """
-        if self.num_points > len(freqs):
+        if self.num_points >= len(freqs):
```

Comment:
@dbochkov-flexcompute not really needed for EME, but should we make this change?